BlueprintUI/Sources/Blueprint View/BlueprintView.swift

```diff
@@ -264,6 +264,12 @@ extension BlueprintView {
         fileprivate func update(node: NativeViewNode, appearanceTransitionsEnabled: Bool) {
             
             assert(node.viewDescription.viewType == type(of: view))
+            
+            // Bail out fast if we do not have any children to manage.
+            // This is a performance optimization for leaf elements.
+            guard self.children.isEmpty == false && node.children.isEmpty == false else {
```

Comment:
Note: We could also optimize single child elements, but I didnt do that here. It'd be a good followup, as many items are single child like Inset.